misc/db_schema.yaml

```diff
@@ -1341,6 +1341,22 @@ route:
     - { Field: ipRouteMask, Type: varchar(256), 'Null': false, Extra: '' }
   Indexes:
     device: { Name: device, Columns: [device_id, context_name, ipRouteDest, ipRouteNextHop], Unique: false, Type: BTREE }
+router_utilization:
+  Columns:
+    - { Field: id, Type: int(11), 'Null': false, Extra: auto_increment }
```

Comment:
Can you explain these columns please?

Reply:
Hey,

So I guess `id` and `device_id` are self explanatory :)
As for storing the OIDs, it seemed easier for me to have them in DB, then loop on all my columns to match the DB OID with the SNMP table that was just fetched (Arista uses DisplayString in the MIB and it was painful without storing the OID).
`resource`, `feature` and `forwarding_element` are straight from Arista doc but it still is multi-vendor compliant :  resource is a "raw" hardware element/memory on the router (typically a TCAM, LEM, ... any kind of "memory") storing data for a feature (ip routing, ACLs, MPLS, ARP...) - and you can have multiple chips (forwarding elements) in the box.
`current` and `maximum` allows us to compute free space - I preferred to keep the maximum rather than only storing the computed free space because you can sometimes carve the memory on devices, thus the maximum can change (and it can be nice to have its value over time).